docs/robots/husky/user_manual_husky.mdx

```diff
@@ -24,44 +31,51 @@ Contained in your Husky shipment are the following items:
 - Lockout Keys (2X)
 - Power Connectors (3X)
 
-### What's required
-
-To use Husky, you will require a PC capable of running Robot Operating System (ROS).
-If a computer option was purchased from Clearpath Robotics with Husky, then the included laptop or minicomputer has
-already been configured with Ubuntu 20.04 and ROS Noetic, plus appropriate ROS packages and other configuration for use with Husky.
+If you purchased standard payload modules or custom integration services with Husky, then additional
+equipment will be included per your specific configuration, plus further documentation as required.
 
-### Expansions
+### Hardware Overview
 
-To expand the capabilities of Husky, consider the following accessories offered by Clearpath Robotics:
+#### System Architecture
 
-- Spare Battery
-- IMU
-- LIDAR
-- Network Camera
-- GPS
+Dingo is built around a computer running Ubuntu (Intel-based computer or Jetson developer kit), paired
+with a 32-bit MCU. The MCU handles I/O, power supply monitoring, and motor control, as well as supplying
+data from the integrated IMU. The communication channel between the MCU and computer is a serial connection.
```

Comment:
I don't think the Husky has an IMU on the circuit board. We frequently add a UM7 inside the user bay, but it does not come with the base robot.

Reply:
Updated